src/index.tsx

```diff
@@ -0,0 +1,236 @@
+        mx: any,
+        my: any;
+
+      if (canvas.offsetParent !== undefined) {
+        do {
+          offsetX += canvas.offsetLeft;
+          offsetY += canvas.offsetTop;
+        } while ((canvas = canvas.offsetParent));
+      }
+
+      if (e.touches) {
+        mx = (e.clientX || e.touches[0].clientX) + offsetX;
+        my = (e.clientY || e.touches[0].clientY) + offsetY;
+      } else {
+        mx = e.clientX + offsetX;
+        my = e.clientY + offsetY;
+      }
+
+      return { x: mx, y: my };
+    }
+
+    function handleMouseDown(e: any) {
+      isDrawing = true;
+      lastPoint = getMouse(e, canvas);
+      prevjob = { done: false, x: 0, y: 0 };
+    }
+
+    var pstate: any;
+    var plvalx: any;
+    var plvaly: any;
+
+    function handleMouseMove(e: any) {
+      if (!isDrawing) {
+        return;
+      }
+
+      e.preventDefault();
+
+      if (this.state.choice === "rectangle") {
+        if (prevjob.done === false) {
+          prevjob.done = true;
+          pstate = canvas.toDataURL();
+        }
+        canvasPic = new Image();
+        canvasPic.src = pstate;
+        canvasPic.onload = () => {
+          if (e.touches) {
+            plvalx = e.clientX || e.touches[0].clientX;
+            plvaly = e.clientY || e.touches[0].clientY;
+          } else {
+            plvalx = e.clientX;
+            plvaly = e.clientY;
+          }
+          ctx.drawImage(canvasPic, 0, 0);
+          ctx.fillStyle = this.state.background;
+          ctx.beginPath();
+          ctx.rect(
+            lastPoint.x,
+            lastPoint.y,
+            plvalx - lastPoint.x,
+            plvaly - lastPoint.y
+          );
+          ctx.fill();
+          return;
+        };
+        return;
+      }
+
+      if (this.state.choice === "circle") {
+        if (prevjob.done === false) {
+          prevjob.done = true;
+          pstate = canvas.toDataURL();
+        }
+        canvasPic = new Image();
+        canvasPic.src = pstate;
+        canvasPic.onload = () => {
+          if (e.touches) {
+            plvalx = e.clientX || e.touches[0].clientX;
+            plvaly = e.clientY || e.touches[0].clientY;
+          } else {
+            plvalx = e.clientX;
+            plvaly = e.clientY;
+          }
+          ctx.drawImage(canvasPic, 0, 0);
+          ctx.fillStyle = this.state.background;
+          ctx.beginPath();
+          ctx.arc(
+            lastPoint.x,
+            lastPoint.y,
+            Math.max(
+              Math.abs(plvalx - lastPoint.x),
+              Math.abs(plvaly - lastPoint.y)
+            ),
+            0,
+            2 * Math.PI
+          );
+          ctx.fill();
+          return;
+        };
+        return;
+      }
+
+      var currentPoint: any = getMouse(e, canvas),
+        dist: any = distanceBetween(lastPoint, currentPoint),
+        angle: any = angleBetween(lastPoint, currentPoint),
+        x: any,
+        y: any;
+
+      for (var i: any = 0; i < dist; i++) {
```

Comment:
No need to specify any type here. Just leave it as it is, it's clear that it is a number